C_CPP/Algorithmic Approaches/Backtracking/m-coloring/m.txt

```diff
@@ -0,0 +1,5 @@
+4

```

Comment:
Delete this file m.text, you've already included input in your code and remove it from readme as well